src/environment_setup/python_package.ml

```diff
@@ -72,5 +72,5 @@ let default ~host ~run_program ~install_path () =
     create_python_tool ~host ~run_program ~install_path
       ~version:"0.0.21" (Pip, Package_PyPI "topiary");
     create_python_tool ~host ~run_program ~install_path
-      ~version:"0.3.1" (Pip, Package_PyPI "vaxrank");
+      ~version:"0.4.0" (Pip, Package_PyPI "vaxrank");
```

Comment:
I'm not sure what the reasoning was; looking for @iskandr input here. Why didn't we do this before for e.g.? If it's not a default in Vaxrank (since it's by default off), I'd lean towards leaving it that way in general (what's actually run in Epidisco is a different matter, though; we can default to whatever we'd like there, I'm just not sure what the reasoning is for going one way over another).